dev/benchmarks/platform_channels_benchmarks/lib/main.dart

```diff
@@ -161,6 +161,7 @@ Future<void> _runTest({
   required String name,
   required int numMessages,
 }) async {
+  print('running $name');
```

Comment:
I think it will be useful.  It only prints out about 10 things.  It's especially useful when running locally since the benchmarks can leave the console inactive for multiple minutes.